cogeo_mosaic/backends/base.py

```diff
@@ -64,3 +69,82 @@ def write(self):
     @abc.abstractmethod
     def update(self):
         """Update existing MosaicJSON on backend."""
+
+    def _update_quadkey(self, quadkey: str, dataset: List[str]):
```

Comment:
Is there a situation where the user wants to update a single quadkey?  If not I don't think we need this method.

Reply:
this was made to accommodate DyanmoDB where we need to update each quadkey